internal/app/collector/repositories/validators.go

```diff
@@ -0,0 +1,66 @@
+package repositories
```

Comment:
do you think it's better to come up with another package name to avoid confounding with the standalone repositories library? if yes, suggest one please